contentcuration/contentcuration/templatetags/license_tags.py

```diff
@@ -1,11 +1,11 @@
+from contentcuration.models import License
 from django import template
 from django.template.defaultfilters import stringfilter
 from django.utils.translation import ugettext_lazy as _
 
-from contentcuration.models import License
 register = template.Library()
 
-LICENSE_MAPPING = {l.license_name: l.license_url for l in License.objects.all()}
+LICENSE_MAPPING = None
```

Comment:
This change was needed to avoid having to connect to the postgres server during collectstatic.